File: boletim-ocorrencia/src/main/java/br/edu/utfpr/td/tsi/sistema/boletim/ocorrencia/model/Vehicle.java
package br.edu.utfpr.td.tsi.sistema.boletim.ocorrencia.model;

public class Vehicle {

    private Integer year;
    private String color;
    private String brand;
    private String type;
    private Registration registration;

    public Integer getYear() {
        return year;
    }

    public String getColor() {
        return color;
    }

    public String getBrand() {
        return brand;
    }

    public String getType() {
        return type;
    }

    public Registration getRegistration() {
        return registration;
    }

    public void setYear(Integer year) {
        this.year = year;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public void setType(String type) {
        this.type = type;
    }

    public void setRegistration(Registration registration) {
        this.registration = registration;
    }
}

File: boletim_python_v/occurance_report_system/occurance_report_system/occurance_report/models/vehicle.py
from .registration import Registration


class Vehicle:
    def __init__(self):
        self.year = None
        self.color = None
        self.brand = None
        self.type = None
        self.registration = None

    def get_year(self):
        return self.year

    def get_color(self):
        return self.color

    def get_brand(self):
        return self.brand

    def get_type(self):
        return self.type

    def get_registration(self):
        return self.registration

    def set_year(self, year):
        self.year = year

    def set_color(self, color):
        self.color = color

    def set_brand(self, brand):
        self.brand = brand

    def set_type(self, type):
        self.type = type

    def set_registration(self, registration):
        self.registration = registration
